.github/workflows/CI.yml

```diff
@@ -0,0 +1,84 @@
+name: CI
+# Run on master, any tag or any pull request
+on:
+  push:
+    branches: [master]
+    tags: ["*"]
+  pull_request:
```

Comment:
Does this not need values after the `:`?

Reply:
I think so... because this needs to be dict with the `push` and `pull_request` keys rather than a list